src/openforms/submissions/tests/test_variables/test_num_queries.py

```diff
@@ -57,9 +57,10 @@ def test_evaluate_form_logic_without_rules(self):
         # 1. get_dynamic_configuration: injects variables in configuration and *currently* does 1 query to retrieve
         #    variables with prefill data, so the defaultValue on the components can be set.
         # 2. Retrieve all logic rules related to a form
-        # 3. Load submission state: Retrieve formsteps,
-        # 4. Load submission state: Retrieve submission steps
-        with self.assertNumQueries(4):
+        # 3. Retrieve auth info for the submission
```

Comment:
For this, we need the submission object passed to `evaluate_form_logic` to already have the auth_info. I added the `auth_info` prefetch on the `get_object` of the endpoint (src/openforms/submissions/api/viewsets.py#L361).

But since this only tests `evaluate_form_logic` Im not sure if I should remove the query from the assertions?